pkg/cmd/util/output/backup_describer.go

```diff
@@ -238,6 +238,22 @@ func DescribeBackupStatus(d *Describer, backup *velerov1api.Backup, details bool
 	d.Printf("Expiration:\t%s\n", status.Expiration)
 	d.Println()
 
+	if backup.Status.Progress != nil {
+		if backup.Status.Phase == velerov1api.BackupPhaseInProgress {
+			// if the backup's in progress, these two numbers are most meaningful
+			// for showing progress, and they avoid getting into the nuances of
+			// "items processed" vs. "items backed up".
+			d.Printf("Total items to be backed up:\t%d\n", backup.Status.Progress.TotalItems)
+			d.Printf("Items backed up so far:\t%d\n", backup.Status.Progress.ItemsBackedUp)
+		} else {
+			// if no longer in progress, just show total # of items backed up since
+			// we just want to summarize what was done.
```

Comment:
Maybe unpopular opinion :) : If it was me, and you ***are*** giving me these stats, then I'd also want to see the predicted number of items to be backed up, so I can tell if you really backed up all of it. Basically I want to know it backed up 100% of the items. Obviously if the backup was successful this is implied, but I'm thinking if we could help inform that in this context. Maybe adding something like "100% of the items backuped" would be helpful?

Reply:
Yeah I think that makes sense, this was more relevant when I had the three different fields and was attempting to avoid showing mismatched numbers side-by-side to avoid confusion. I think I'll just leave both fields, even if it's completed, I may just relabel the one to not say "estimated" since by the time we're done, we know exactly how many there were.

Reply:
Yes, perfect.